README.md

```diff
@@ -1,7 +1,7 @@
 ![Dronecode SDK](https://github.com/dronecore/sdk_docs/raw/develop/assets/site/sdk_logo_full.jpg)
 
-[![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=DroneCore/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/DroneCore/activity)
-[![travis-ci build status](https://travis-ci.org/dronecore/DroneCore.svg?branch=develop)](https://travis-ci.org/dronecore/DroneCore)
+[![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=DronecodeSDK/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/DronecodeSDK/activity)
+[![travis-ci build status](https://travis-ci.org/Dronecode/DronecodeSDK.svg?branch=develop)](https://travis-ci.org/Dronecode/DronecodeSDK)
 [![appveyor build status](https://ci.appveyor.com/api/projects/status/1ntjvooywpxmoir8/branch/develop?svg=true)](https://ci.appveyor.com/project/julianoes/dronecore/branch/develop)
```

Comment:
Looks good to me. Feel free to merge. Only question, why an @julianoes appveyer branch? https://ci.appveyor.com/project/julianoes/dronecore/branch/develop